### DISCUSSION TITLE
Question about the training_reuse_numb_steps

### DISCUSSION THREAD

**Discussion by xiongluojie** [deepmodeling/dpgen#1600] [Q&A]
When I continue training a model from the last iteration, what is the appropriate value for training_reuse_numb_steps? Can it be significantly fewer than the number of steps used in the initial training of the model?
Thanks!

**Comment by wanghan-iapcm** [comment]
may not always be true. the new data (data labeled by the previous iteration) should be properly trained with `training_reuse_numb_steps` training steps.

**Reply by xiongluojie** [reply]
I see. Thank you for your help!
